simpleflu/README.md

```diff
@@ -1,28 +1,23 @@
 # Simple Flu
 
-This example introduces the **Simple Flu** model, which is a standalone model of flu infection and the basis for the `../flu-with-behavior`, `../school-closure`, and `../vaccine` tutorials.
+This example introduces the **Simple Flu** model, a standalone model for influenza and the basis for the `../flu-with-behavior`, `../school-closure`, and `../vaccine` tutorials.
+
 
 ## Introduction
 
-This model defines a condition, `INFLUENZA`, which agents can contract via either the meta agent (to create initial infections) or other agents.
-Having contracted the condition, agents either do or do not develop symptoms and then recover, at which point they are immune
-and can no longer contract `INFLUENZA`.
+This model defines a condition, `INFLUENZA`, which agents can contract either via the meta agent (a source of initial infections) or interaction with other agents. Having contracted the condition, agents may develop symptoms or remain astmptomatic before recovering, at which point they are immune and can no longer contract `INFLUENZA`.
+
 
 ## Review of code implementing the model
 
-The code that implements the **Simple Flu** model is contained in two `.fred` files:
+The code that implements the **Simple Flu** model is contained in two **`.fred`** files:
 
-- `main.fred`
-- `simpleflu.fred`
+- **`main.fred`**
```

Comment:
using the filename formatting that is used throughout the guide.  In rst files, this is the same as filename:`<file_name>`.